### ISSUE TITLE
ng2-simple-autocomplete is not a known element

### ISSUE THREAD

**Issue by ChaudhryAtif** [rhostem/ng2-simple-autocomplete#2]
After installing, and importing, when I try to follow the examples and paste the code in the html, I get the following error:

`Angular: 'ng2-simple-autocomplete' is not a known element. 1. If 'ng2-simple-autocomplete' is an Angular component, then verify that it is part of this module. 2. If 'ng2-simple-autocomplete' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message.`

![image](https://user-images.githubusercontent.com/7082580/31923751-18dcae64-b842-11e7-8a65-a0b91cdb4c8f.png)

**Comment by ChaudhryAtif** [comment]
@rhostem any suggestion?

**Comment by rhostem** [comment]
Sorry I couldn’t  figure out exact problem of this issue. It looks like something related to AOT compilation and angular-cli.

https://github.com/angular/angular/issues/15890

So I’m going to version up this repo for AOT compatibility with [angular-library-seed](https://github.com/trekhleb/angular-library-seed) guided by [‘How to create AOT/JIT compatible Angular 4 library with external SCSS/HTML templates
‘](https://medium.com/@trekhleb/how-to-create-aot-jit-compatible-angular-4-library-with-external-scss-html-templates-9da6e68dac6e). But it will take a time… 

If you’d like to use this component in your app now, copying original sources from `src/ng2-simple-autocomplete` folder will be temporary solution.

And this component works well with [angular-starter](https://github.com/AngularClass/angular-starter) using webpack, not using angular cli. (But build failed in AOT mode. and succeed in normal build)

**Comment by ChaudhryAtif** [comment]
@rhostem When do you expect to get an update released?

**Comment by rhostem** [comment]
I'm not sure. Becuase I dont have focus on Angular for now.

**Comment by ChaudhryAtif** [comment]
@rhostem Alright. In the meantime, I should be able to copy index.ts and model.ts files in [src/ng2-simple-autocomplete](https://github.com/rhostem/ng2-simple-autocomplete/tree/master/src/ng2-simple-autocomplete) and use it as instructed?

**Comment by rhostem** [comment]
Yes. then add the component to 'declarations' array of your module.
